app/models/commit.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+# A specific commit belongs to a repository and can have many file versions.
+class Commit < Sequel::Model
+  plugin :timestamps
+
+  many_to_one :repository
+  one_to_many :file_version
+  many_to_one :author, key: :author_id, class: :User
+  many_to_one :editor, key: :editor_id, class: :User
+end
```

Comment:
We could set the `author`/`editor` when `author_email`/`editor_email` changes. See [the Sequel documentation](http://sequel.jeremyevans.net/rdoc/files/doc/association_basics_rdoc.html#label-_association-3D+-28-3Asetter+option-29) on how to do this. I'm not sure if the documentation is applicable on this case because we want to intercept a setter of an attribute and not a setter of an association, but it may work here as well.

This should be reflected in the tests ;)